docs/api/05_subscriptions/create-subscription.mdx

```diff
@@ -131,23 +131,21 @@ import TabItem from '@theme/TabItem';
 ```json
 {
   "subscription": {
-    "customer_id": "5eb02857-a71e-4ea2-bcf9-57d3a41bc6ba",
+    "external_customer_id": "5eb02857-a71e-4ea2-bcf9-57d3a41bc6ba",
     "plan_code": "new_code",
-    "unique_id": "id",
     "name": "name",
-    "subscription_id": "sub id",
+    "external_id": "sub id",
     "billing_time": "anniversary"
   }
 }
 ```
 
 | Attributes | Type | Description |
 | -----------| -----| ----------- |
-| customer_id | String &nbsp &nbsp &nbsp<Required>**Required**</Required> | Customer unique identifier in your application |
+| external_customer_id | String &nbsp &nbsp &nbsp<Required>**Required**</Required> | Customer unique identifier in your application |
 | plan_code | String &nbsp &nbsp &nbsp<Required>**Required**</Required> | Code identifying the plan.<br></br>It must match the `code` property of one of the plans. |
-| unique_id | String &nbsp &nbsp &nbsp<Optional>**Optional**</Optional> | Unique external identifier of the subscription. Used as an idempotency key. Can also be used for upgrades/downgrades. Default value if not provided: `customer_id`. |
+| external_id | String &nbsp &nbsp &nbsp<Optional>**Optional**</Optional> | Unique external identifier of the subscription. Used as an idempotency key. Can also be used for upgrades/downgrades. Default value if not provided: `external_customer_id`. |
```

Comment:
What happens if the user creates several subscriptions for the same customer without providing any `external_id`? I understand that for the first subscription, it will default to `external_customer_id` - what about the next subscriptions?

Reply:
That's a good question - Indeed, for the first subscription, it will default to `external_customer_id`.
For the following, it won't create the subscription, but return the one previously created

Reply:
Thanks, @rsempe! If I understand correctly, this could lead to the following situation:

1. I create a subscription, which is linked to plan A, without providing any `external_id` so it defaults to `external_customer_id`; and
2. If later I want to create another subscription, which is linked to plan B, but don't provide an `external_id`, it will trigger an upgrade (or downgrade) instead of creating a new subscription.

Is this correct?

cc @jdenquin (as discussed during the roadmap catch-up)

Reply:
Maybe I'm wrong but I was thinking that we generate an external_id if it's not provided, but maybe it's here only for the not multiple plan?